src/AccessLog.php

```diff
@@ -126,7 +126,11 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
             $context = $contextFunction($request, $response);
         }
 
-        if ($response->getStatusCode() >= 400 && $response->getStatusCode() < 600) {
+        $statusCode = $response->getStatusCode();
+
+        if ($statusCode >= 400 && $statusCode < 500) {
+            $this->logger->warning($message, $context);
```

Comment:
Maybe this should even be `notice` instead of `warning`...

Reply:
Warning is more acurate 👍